Sources/ArrayFile.swift

```diff
@@ -27,12 +27,16 @@
 
 import Foundation
 
-/// A representation of a filesystem array file.
-///
-/// The data type is NSArray.
+/**
+ A representation of a filesystem array file.
+
+ The data type is NSArray.
+*/
 public typealias NSArrayFile = File<NSArray>
```

Comment:
Is it a swift4 needs to change all the comments?
If not, a separated PR would have been better (to see only swift 4 change, and discuss elsewhere about comment formatting)

Reply:
Swift 4 _does not_ require a change in comment style. I've updated [RandomKit](https://github.com/nvzqz/RandomKit) to Swift 4 and much has remained the same as before, including the comment style.